src/components/AccountDetails/index.tsx

```diff
@@ -130,11 +129,12 @@ const AccountControl = styled.div`
 `
 
 const AddressLink = styled(ExternalLink)`
-  font-size: 0.825rem;
   color: ${({ theme }) => theme.deprecated_text3};
   margin-left: 1rem;
   font-size: 0.825rem;
   display: flex;
+  gap: 6px;
```

Comment:
if this is a shared component, make sure this doesn't affect other places where it is (same with the deletion of font)

Reply:
the font was duplicated